elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -143,10 +143,6 @@ def find_bugs_sweep_cli(runtime: Runtime, advisory_id, default_advisory_type, ch
         logger.info('No bugs found')
         sys.exit(0)
 
-    if output == 'text':
```

Comment:
I don't know in which case this would be useful, in sweep we're printing out bugs by each category. In --use-default-advisory case it lists all bugs found which is confusing, better to remove it

Reply:
```
-o, --output [text|json|slack]  Applies chosen format to --report output
```
I guess it should be `if report and output == "text":
```